peachjam/forms.py

```diff
@@ -232,9 +244,21 @@ def send_email(self):
             "app_name": settings.PEACHJAM["APP_NAME"]
         }
 
-        mail_admins(
+        # get admin emails from settings
+        admin_emails = [admin[1] for admin in settings.ADMINS]
+
+        # get admin emails from site settings
+        site_admin_emails = list(
+            pj_settings().admin_emails.values_list("email", flat=True)
+        )
+
+        recipients = admin_emails + site_admin_emails
+
+        email = EmailMultiAlternatives(
```

Comment:
`send_mail` will do all this for you. It's very similar to the `mail_admins` that was there previously, you just need to add `recipient_list=...`.